R/mod_marg.R

```diff
@@ -52,6 +64,20 @@ mod_marg2 <- function(mod, var_interest,
     all(names(at) %in% names(data))
   )
 
+  if(is.null(dof) & !is.null(vcov_mat) & mod$family$family == 'gaussian')
+    warning(paste(
+      "You provided a new variance-covariance matrix for an OLS model",
```

Comment:
Changed to be a bit shorter and reference the details in the documentation. I think the warning is useful, because it's not something people think about, and the estimates / SE's replicate fine so it's easy to miss the incorrect P value.